example/pubspec.lock

```diff
@@ -57,6 +57,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.0.3"
+  device_frame:
```

Comment:
Should this be excluded from history or added to .gitignore?

Reply:
yeah its been annoying dealing with it locally I should probably add it